nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-standard-record-utils/src/main/java/org/apache/nifi/schema/access/ConfluentSchemaRegistryStrategy.java

```diff
@@ -67,7 +67,7 @@ public RecordSchema getSchema(final Map<String, String> variables, final InputSt
         final int schemaId = bb.getInt();
 
         final SchemaIdentifier schemaIdentifier = SchemaIdentifier.builder()
-                .id(Long.valueOf(schemaId))
+                .schemaVersionId(Long.valueOf(schemaId))
                 .version(1)
```

Comment:
I'm wondering if we can also remove this hard-coded version 1, I don't think it was even used for anything before, but now that we are switching to `schemaVersionId`, it should definitely not be used.

Reply:
yep, that's fair, I don't see this version 1 used anywhere - I pushed a commit to remove it